templates/email/webform-email-html.html.twig

```diff
@@ -0,0 +1,24 @@
+{#
+/**
+ * @file
+ * Default theme implementation for webform email wrapper template as HTML.
+ *
+ * @ingroup themeable
+ */
+#}
+<html>
+	<head>
+		<title>{{ subject }}</title>
+	</head>
+	<body>
+		<div id="header" style="background-color: #880088; padding: 10px 0px 0px 10px;">
+			<div id="container" style="padding-left:50px">
+				<img src="https://www.croydon.gov.uk/themes/contrib/localgov_theme_croydon/logo.svg" alt="Croydon Council Forms Site Logo" width="154px" height="56px">
```

Comment:
This  alt text should be a translatable string

{{ 'Croydon Council Forms Site Logo'|t }}